org.skytemple.SkyTemple.yml

```
@@ -0,0 +1,134 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=ipc"
+  - "--filesystem=host"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+cleanup:  # todo
+  - /bin/pcap-config
+  - /include
+  - /lib/libpcap.a
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: cmake-ninja
+    no-make-install: true
+    build-options:
+      cxxflags: "-DCMAKE_BUILD_TYPE=Release"
```

```suggestion
    config-opts:
      - -DCMAKE_BUILD_TYPE=Release
```

nope. this is a cmake option.

---

Hm, figured as much was hoping this would just get passed down. But I couldn't find a way to set cmake options in the reference, what option is that? Does `make-args` do it for CMake builds?

---

whatever I suggested is what you want.

---

`make-args `is usually targets. It is passed to `ninja` or `make` (it would be `ninja` in that case)

---

Oh right, you suggested changes :man_facepalming: